www-apps/gitea/gitea-1.7.1.ebuild

```diff
@@ -8,19 +8,20 @@ EGO_PN="code.gitea.io/gitea"
 KEYWORDS="~amd64 ~arm"
 
 DESCRIPTION="A painless self-hosted Git service, written in Go"
-HOMEPAGE="https://github.com/go-gitea/gitea"
+HOMEPAGE="https://gitea.io/"
 SRC_URI="https://github.com/go-gitea/gitea/archive/v${PV}.tar.gz -> ${P}.tar.gz"
 
 LICENSE="MIT"
 SLOT="0"
+IUSE="sqlite pam"
 
 DEPEND="
 	dev-go/go-bindata
 	sys-libs/pam
```

Comment:
Double depend on pam?

Reply:
Yes: pam is required during the build (along with `dev-go/go-bindata`) and is also required at runtime, but not `dev-go/go-bindata`.